### ISSUE TITLE
Callback response should be standardized

### ISSUE THREAD

**Issue by devdazed** [mattrobenolt/node-snsclient#3]
The community standard for callbacks in node is "callback(error, data)".  Currently you callback with the message or an error in the first parameter which forces people to do type checking to see if something wen't wrong or not.

**Comment by mattrobenolt** [comment]
Oops, you are right. Any clever way I can fix this without making a backwards incompatible change?

**Comment by devdazed** [comment]
not really, because most people do something like this:

``` js
doStuff(function(err, res){
  if(err){
    throw(err);
  }

  //do stuff
});
```

so as you can see, there will need to be an API change. Maybe it can be solved with a line in the `README.md` that specifies the API change.

**Comment by mattrobenolt** [comment]
Right. I built this, I think before that format was really standardized. Or I was too new to Node at the time to _know_ of the standard. :)

I guess I can fix it up, bump to 0.2, and put a notice in the README.

Thanks!

**Comment by devdazed** [comment]
no problem, thanks for making this package! :)

**Comment by mattrobenolt** [comment]
Glad somebody found it useful! :) I'll get this fixed up today.

**Comment by mattrobenolt** [comment]
@devdazed Fixed and pushed 0.2.0 to npm.

Thanks!

**Comment by devdazed** [comment]
awesome, thanks!
